docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+
+Below we list the typical roles across an organizing committee. We note that for some events, certain roles might not be considered an integral part of the formal organizing committee, for example the tutorial leads. However, we believe there are several advantages to including these people at the decision making level of the overall event. These include an increased sense of ownership that results in organizers putting forth their best effort in designing and delivering content, and a broadening of the collective wisdom of the organizing committee around important decisions due to the presence of a diversity of perspectives. Early career faculty and researchers also often report that being recognized at the organizational level helps their resumes when it comes to applying for jobs. 
+
+## Tutorial Leads
+
+Generally the largest group of organizers are focused around development and delivery of data science tutorials. Responsibilities include:
+
+* polling their respective communities to determine what are the community needs around data science education
+* talking to open source software developers to connect with the most up-to-date tools and libraries available
+* building a set of learning objectives, designing and creating a tutorial
+* review the content of a tutorial and integrate it with themes of other tutorials offered in the event
+* work with cloud computing organizers to ensure all the tools are in place for participants to run interactive tutorials
+* practice the delivery of tutorials in advance of the event
+
+Additional guidelines on the roles and responsibilities for tutorial leads can be found [here](https://geohackweek.github.io/wiki/tutorial_leads.html).
+
+## logistics coordinator
+
+The logistics coordinator plays an important role in event planning and implementation. A partial list of their responsibilities has been compiled [here](https://github.com/geohackweek/administrative/issues) as a set of GitHub issues. (ask Jane for one of her checklists)
+
+## Project/Hack coordinator
+
+A project/hack coordinator helps to guide the formation and smooth functioning of projects and hacking during the event. Responsibilities include:
+
+* guiding a team in defining project goals that can be reasonably accomplished during the week
+* setting benchmarks for team activities and checking-in regularly with team members on progress
+* helping to ensure that everyone gets a chance to participate in group work in some way
+* guide the team in populating GitHub project repositories and README files following the Project Guidelines
+* enabling and encourage participants to solve problems in a way that builds confidence
+* supporting the project scientist in establishing workflows on GitHub
+* mentoring individual team members requesting additional support
+* working with the team to navigate technical or team dynamic challenges

```

Comment:
I feel like this list is maybe a bit too Geo Hack Week specific? For example, Astro Hack Week doesn't have a shared GitHub org where people put their projects, and putting projects on GitHub isn't a requirement. We also don't have project scientists. :)